src/lime/graphics/Image.hx

```diff
@@ -370,7 +370,11 @@ class Image
 			case FLASH:
 				var srcChannel = switch (sourceChannel)
 				{
-					case RED: 1;
+					case 
+				
+				
+				
+				: 1;
```

Comment:
Everything else looks good, but please fix this change that you made by mistake.